.github/workflows/monero-book.yml

```diff
@@ -0,0 +1,30 @@
+# This action attempts to build the Monero book, if changed.
+
+name: Monero mdBook
+
+on:
+  push:
+    paths:
+      - 'books/protocol/**'
+
+env:
+  # Version of `mdbook` to install.
+  MDBOOK_VERSION: 0.4.36
```

Comment:
`monero-book` uses `svgbob` which needs to be installed as well: https://github.com/Cuprate/monero-book/blob/main/.github/workflows/mdbook.yml